src/devices/omni.c

```diff
@@ -0,0 +1,242 @@
+/** @file
+`   Omni Multisensor
+    v1.2
+
+    Copyright (C) 2025 H. David Todd <hdtodd@gmail.com>
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+
+/* clang-format off */
```

Comment:
remove the clang-format here and at the end.
the first line should end with a full-stop